var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -266,10 +266,11 @@ class Openmpi(AutotoolsPackage):
     if sys.platform != 'darwin':
         depends_on('numactl')
 
-    depends_on('autoconf', type='build', when='@develop')
-    depends_on('automake', type='build', when='@develop')
-    depends_on('libtool',  type='build', when='@develop')
-    depends_on('m4',       type='build', when='@develop')
+    depends_on('autoconf', type='build', when='@master')
+    depends_on('automake', type='build', when='@master')
+    depends_on('libtool',  type='build', when='@master')
+    depends_on('m4',       type='build', when='@master')
+    depends_on('pandoc', type='build', when='@master')
```

Comment:
Just leaving this tip here for your information. Since a month or so you can group together common parts of a `when=` argument like:
```python
with when('@master'):
    depends_on('autoconf', type='build')
    depends_on('automake', type='build')
    depends_on('libtool', type='build')
    depends_on('m4', type='build')
    depends_on('pandoc', type='build')
```